layouts/shortcodes/card.html

```diff
@@ -1,49 +1,44 @@
-<div class="td-card card mb-4">
-  {{ with $.Get "header" }}
-    <div class="card-header">
-      {{ if eq $.Page.File.Ext "md" }}
-        {{ $.Get "header" | markdownify }}
-      {{ else }}
-        {{ $.Get "header" | htmlUnescape | safeHTML }}
-      {{ end }}
+{{/* Check parameter type */ -}}
+{{ with .Get "code" -}}
+  {{ if ne ( printf "%T" . ) "bool" -}}
+    {{ errorf "shortcode card: parameter 'code' must be either true or false" -}}
+  {{ end -}}
+{{ end -}}
+
+{{ $code := default false (.Get "code") -}}
+{{ $lang := default "" (.Get "lang") -}}
+{{ $highlight := default "" (.Get "highlight") -}}
+
+<div class="td-card card border me-4">
```

Comment:
As I mentioned before, I'd suggest adding custom styling through `cards-pace.scss`; avoid the `border` class here, and even consider moving the `me-4` into, say, `.td-card.card`.

Reply:
> As I mentioned before, I'd suggest adding custom styling through `cards-pace.scss`; avoid the `border` class here, and even consider moving the `me-4` into, say, `.td-card.card`.

I agree. I will open a new issue that addresses this issue shortly.